src/ua/hillel/FindWordPosition.java

```diff
@@ -0,0 +1,17 @@
+package ua.hillel;
+
+public class FindWordPosition {
+    public static void main(String[] args) {
+
+        findWordPosition();
+    }
+
+    public static void findWordPosition() {
```

Comment:
Your method should return `int` which is a starting position of a target word.
Method `findWordPosition()` should accept 2 parameters: `String source, String target'